README.md

```diff
@@ -1945,11 +1956,14 @@ Some points on negotiating compensation:
   agree what those could look like *if* you demonstrate outstanding performance and the
   company has money.
 - If you’re moving from an established company to a startup, you may be asked to take a
-  salary cut. This is reasonable, but it’s wise to discuss explicitly how much the cut is,
-  and when your salary will be renegotiated.
-  For example, you might take 25% below your previous salary, but there can be an agreement
-  that this will be corrected if your performance is strong and the company gets funding.
-- 🔹 Always negotiate non-compensation aspects before agreeing to an offer.
+  [salary cut](http://fortune.com/2016/06/13/new-job-pay-cut-leadership/). This is reasonable,
+  but it’s wise to discuss explicitly how much the cut is, and when your salary will be
+  renegotiated. For example, you might take 25% below your previous salary, but there can be
+  an agreement that this will be corrected if your performance is strong and the company
+  gets funding.
+- 🔹 Always negotiate
+  [non-compensation aspects](http://time.com/money/4391908/job-offer-negotiations/) before
```

Comment:
Much longer list of negotiable benefits here: http://www.kennedyexecutive.com/blog/salary-negotiation-33-things-negotiate-money/
More specific advice on how to negotiate non-compensation aspects here: https://www.nerdwallet.com/blog/loans/student-loans/negotiate-nonsalary-benefits-job-perks/

Reply:
Added both links (and removed the one I had).